src/flags.rs

```diff
@@ -180,42 +177,40 @@ impl<'this> RustcCodegenFlags<'this> {
         // Flags shared between clang and gnu
         if clang_or_gnu {
             // https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-mbranch-protection
+            // https://gcc.gnu.org/onlinedocs/gcc/AArch64-Options.html#index-mbranch-protection (Aarch64)
+            // https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html#index-mbranch-protection-1 (ARM)
+            // FIXME(madsmtm): Parse the value, and make sure we pass it in the correct format.
             if let Some(value) = self.branch_protection {
                 push_if_supported(
                     format!("-mbranch-protection={}", value.replace(",", "+")).into(),
                 );
             }
             // https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-mcmodel
+            // https://gcc.gnu.org/onlinedocs/gcc/Option-Summary.html (several archs, search for `-mcmodel=`).
+            // FIXME(madsmtm): Parse the model, to make sure we pass the correct value (depending on arch).
```

Comment:
Same for this fixme

Reply:
I'm not sure, but it seems like RISC-V and SPARC all have a few different:
https://gcc.gnu.org/onlinedocs/gcc/SPARC-Options.html#index-mcmodel_003d-6
https://gcc.gnu.org/onlinedocs/gcc/RISC-V-Options.html#index-mcmodel_003d-4

`-mcmodel=medlow`, `-mcmodel=medmid`, `-mcmodel=medany`, `-mcmodel=embmedany`.

I'm not sure how these translate to/from the rustc options?

Reply:
Nan I am not aware that there's such option in rust.

This looks more like something for bios/kernel/bootloader than for regular processes, and it probably only takes effect at link time since it decides the final layout of executable, so maybe we can skip it?

Reply:
I'm fine with ignoring it for now.